cli/src/commands/router/commands/download.ts

```diff
@@ -0,0 +1,132 @@
+import fs from 'node:fs';
+import { Octokit } from 'octokit';
+import { Command, program } from 'commander';
+import pc from 'picocolors';
+import cliProgress from 'cli-progress';
+import decompress from 'decompress';
+import { join } from 'pathe';
+import { BaseCommandOptions } from '../../../core/types/types.js';
+
+export default function (_: BaseCommandOptions) {
+  const command = new Command('download');
```

Comment:
Let's rename the command. It might create confusion between `fetch` and `download`. Both are very similar but do completely different things. Let's call it `wgc router binary download` or `wgc router-download`